spring-cloud-sleuth-zipkin/pom.xml

```diff
@@ -38,6 +38,10 @@
 			<artifactId>spring-cloud-sleuth-core</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-web</artifactId>
```

Comment:
tradeoff here is that this adds a mandatory dependency. The existing class could have been refactored to allow a second reporter to be used (which would make this optional). I don't know the dependency hygiene, but food for thought.